Documentation/APIContracts.md

```diff
@@ -502,16 +502,16 @@ For example:
 ```swift
 producer.start { event in
     switch event {
-    case let .Next(value):
+    case .value(let value):
```

Comment:
Please place `let` before the case name. The RAS codebase has a convention of having value binding pattern indicator "globally" regardless of the number of bound values. :smily